### DISCUSSION TITLE
Search during prior knowledge, set article as relevant, but does not seem to process

### DISCUSSION THREAD

**Discussion by cdnie** [asreview/asreview#660] [Q&A] [answered]
A colleague and I both ran into the same thing during the Prior Knowledge fase. I search for a relevant document, choose Relevant, but it does not seem to process; the results stay at 0. If I use the Random option, and choose Relevant it does process this. We both (apart from each other) have this issue; are we doing something wrong? 
Thank you for your answer!

**Comment by Rensvandeschoot** [comment]
Dear @cdnie ,

Ai, this is unexpected behavior. Sometimes there is an encoding issue causing problems. What software did you use to export the meta-data?

**Reply by cdnie** [reply]
I used EndNote (X9) with the standard export filter Refman(RIS). Thanks :)

**Comment by SagevdBrand** [comment, marked as answer]
Hello @cdnie, 

Thank you for posting this problem!
The answer in #661 also applies to this issue. The issue itself has to do with the LB field as explained in #607 by @J535D165 :

> RIS files with numeric LB field fail on search and review. ASReview classification value is filled with those LB values. Because they are not 0 or 1, items are handled incorrectly in the prior knowledge step (and elsewhere, but hard to get there). 
Workaround: Remove LB fields from RIS file.
